styleguide/package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "start": "start-storybook -p 9001",
     "build-storybook": "build-storybook",
-    "build": "rimraf -rf dist && tsc",
+    "build": "rimraf -rf dist && tsc && cp ./src/components/Logos/*.svg ./dist/src/components/Logos",
```

Comment:
@HenryNguyen5 How should we go about treating svgs in styleguide? `tsc` is not going to carry them over, and there's no webpack either and babel svg loaders treat them as component, so i found plainly copying them over easy for demonstration

Reply:
Hm, should this be in a `assets` dir instead? The assets dir could mirror the same pathing as the `components` folder, and then we could configure the loaders to look in that directory?

Reply:
Bringing in @rupurt to weigh in on this

Reply:
I wonder if we should follow a similar pattern to `material-ui`? It seems like they include the icons in JS so they're part of the build output artifacts. https://github.com/mui-org/material-ui/blob/master/packages/material-ui-icons/src/ZoomIn.js

![material-ui-icons](https://user-images.githubusercontent.com/680789/71955679-d0260300-319d-11ea-8761-9b68d1aa09a0.png)

Could we use something like this in our `build/setup` tasks? https://github.com/kreuzerk/svg-to-ts

Reply:
Sounds like a good solution for now, at least until we can streamline the build process better

Reply:
I tried `svg-to-ts` and `svgr` and ran into issues with both.
Svg-to-ts gets you svg as a string, and what do I do with the string? I cant import them directly, so write components that render them? Render them with dangerous html? They're also generated after the build is done, so the ts components have to go roundabout ways for using those. 
Svgr generates react js components, so they can't be used in ts components. It breaks the import/export patterns too. 
I took a look at what material-ui does, and I understand how they make it work, but without further scripting that build process simple svg processors didn't seem to work, so I'm leaving it as is unless you provide further input

Reply:
You can use the svg string generated by `svg-to-ts` as a data-uri, see here: https://css-tricks.com/lodge/svg/09-svg-data-uris/

I think because we dont edit the svgs very often, we would make this a `prebuild` step to `styleguide`